### DISCUSSION TITLE
REQUEST: Help on how to add a Property Manager Front-End

### DISCUSSION THREAD

**Discussion by UltimateStarCoder** [microrealestate/microrealestate#213] [Ideas]
Hi, I want to add a property manager front end to this project and don't know how to start. Can you give me some directions on how I can create and implement it? We currently only have a Tenant Front-End and Landlord Front-End. We are working on adding functionality to this Open-Source Application for our school's Software Design Project.

**Comment by camelaissani** [comment]
Currently the management of properties can be done in the landlord frontend.
What do you want provide with a dedicated frontend for properties?

**Comment by UltimateStarCoder** [comment]
Hi! Right now, property management happens in the landlord frontend indeed, but we’re looking to create a separate one just for Property Managers. The idea is to keep things simple and specific to their role. Landlords (the property owners) have full control and can assign Property Managers to manage there property. By having a dedicated Property Manager frontend, we can limit their access to just what they need to manage a given property without full admin control on the property infomration.

This keeps the landlord’s settings safe and gives Property Managers a cleaner, more focused interface. Down the line, we also want to add a warranty tracking feature where landlords have full access to create, delete, and manage warranties, while Property Managers can only view and update (but not delete or create). This setup should make it easier for both roles to work together while keeping the landlord's control. That's what some of the features we want to add to this open-source project for our semester task :)

we have a lot more we want to add but that is the first few things for now :)

**Comment by camelaissani** [comment]
I see thanks for the clarification.
Ambitious project 🙂

You might be inspired by the tenant frontend, clone it, remove what is not needed and add the property management features.

**Reply by UltimateStarCoder** [reply]
Thank you 🙂! I will try that and get back to you here if I run into any trouble!

**Reply by csshad** [reply]
Hi, did you manage to create the property manager frontend?

**Reply by Kuki77** [reply]
@UltimateStarCoder 
Have you had success with your idea?!
